Project/Fall2020_CSC403_Project/TextEngine.cs

```diff
@@ -188,16 +211,40 @@ private void HandleMarkup(string line)
                     }
                     this.Options.Peek().OptionFocused = true; //Focus the top option 
                     this.ChangeText(newLineOptions);
+                    options.Reverse();
```

Comment:
I am still not sure about this

Reply:
I think there must be some kind of ambiguity with options naming happening, but if it works it works and I won't harp on it

Reply:
I think since it's in a switch case there's ambiguity in this case from previous case with something named options

Reply:
That could be a problem in the future to look at

Reply:
It is kind of weird but i dont think there are any negative effects to reversing the options list back to the way it was since it has already done its job of filling the stack at this point